docs/content/core/volumes.md

```diff
@@ -213,5 +214,115 @@ fin config set DOCKSAL_VOLUMES=none
 fin project reset
 ```
 
-Use `fin bash` to log into bash and checkout files into `/var/www` with git. Or use `fin docker cp` to copy 
-files into the `cli` container. 
+Use `fin bash` to log into bash and checkout files into `/var/www` with git. Or use `fin docker cp` to copy
+files into the `cli` container.
+
+### Mutagen {#volumes-mutagen}
+
+- This option only makes sense with Docker Desktop on macOS.
+- Like the Unison option, Mutagen is used to synchronize the files from host to the container.
+- There are two configuration and usage options for Mutagen: Docker Desktop Extension and Stand alone.
+
+#### The Docker Desktop Extension
+
+- This option is more recent but comes in two flavors
+- The Free version which only allows for one Cache Setup.
+- The Paid Subscription ( https://mutagen.io/account/subscription $7/month) which allows for multiple cache directory setups.
+- The Docker Desktop Extension provides a nice GUI integration with Docker Desktop and thus the maintenance of it is easier.
+- To use it run
+
+```bash
+fin config set DOCKSAL_VOLUMES=bind
+docker extension install mutagenio/docker-desktop-extension:latest
+docker context use desktop-linux-mutagen
+fin project reset
+```
+
+  - Then create the cache as instructed here: https://mutagen.io/documentation/docker-desktop-extension
```

Comment:
This should be

- Then create the [cache as instructed](https://mutagen.io/documentation/docker-desktop-extension)